lib/utils.py

```diff
@@ -108,6 +107,26 @@ def set_extras_n_fix_units(
 
     args["resource_provides_quoted"] = [fixquote(x) for x in args["resource_provides"]]
 
+    # if the user defined the usage_model on the command line,
+    # we need to use their definition of usage_model, not ours
+    # also, we define the site as 'Fermigrid' if they have not requested OFFSITE
+    add_site = ""
+    for r in args["resource_provides_quoted"]:
+        if "usage_model" in r:
+            args["usage_model"] = ""
+            if "OFFSITE" not in r:
+                add_site = "Fermigrid"
+
+    # if the user chooses 'onsite' from the runtime params
+    # we need to define the sites as 'Fermigrid'
+    if args["usage_model"] is not "" and "OFFSITE" not in args["usage_model"]:
```

Comment:
So I had no idea that `is not` worked with strings, but when I did some investigation, it looks like the python interpreter tries to warn that it's improper.  Would you mind change that to either `args["usage_model"] != ""` or `(not args["usage_model"])`?